pkg/gardenlet/controller/shoot/shoot/reconciler.go

```diff
@@ -822,11 +818,10 @@ func (r *Reconciler) patchShootStatusOperationSuccess(
 	}
 
 	switch v1beta1helper.GetShootETCDEncryptionKeyRotationPhase(shoot.Status.Credentials) {
-	case gardencorev1beta1.RotationPreparing:
-		v1beta1helper.MutateShootETCDEncryptionKeyRotation(shoot, func(rotation *gardencorev1beta1.ETCDEncryptionKeyRotation) {
-			rotation.Phase = gardencorev1beta1.RotationPrepared
-			rotation.LastInitiationFinishedTime = &now
-		})
+	// TODO: Remove rotation prepared case in a future release.
```

Comment:
```suggestion
// TODO(AleksandarSavchev): Remove rotation prepared case in gardener `v1.<version>`
```
Similarly in other occurrences